bitnami/elasticsearch/templates/coordinating-deploy.yaml

```diff
@@ -71,6 +71,8 @@ spec:
               value: {{ .Values.name | quote }}
             - name: ELASTICSEARCH_CLUSTER_HOSTS
               value: {{ include "elasticsearch.hosts" . | quote }}
+            - name: ELASTICSEARCH_TOTAL_NODES
+              value: {{ add .Values.master.replicas .Values.data.replicas | quote }}
```

Comment:
Is it necessary to quote it? I mean, it's an integer, right?

Reply:
Maybe it is not necessary, I just adapted it from
```yaml
            - name: ELASTICSEARCH_MINIMUM_MASTER_NODES
              value: {{ add (div .Values.master.replicas 2) 1 | quote }}
```
I can try without the `quote`

Reply:
Fine then